src/app/plan/duck/operations.ts

```diff
@@ -19,6 +17,9 @@ import {
 } from '../../../client/resources/conversions';
 import { commonOperations } from '../../common/duck';
 import { isSelfSignedCertError, handleSelfSignedCertError } from '../../common/duck/utils';
+import { groupPlan, groupPlans, getMigrationStatus, getPlanStatus, getPlanPVs } from './utils';
```

Comment:
I almost want to import this as `import planUtils from './utils'` so the usage reads: `planUtils.getPlanPVs`. Makes things more clear where they're coming from. What do you think?